android/src/main/java/expo/modules/xmtpreactnativesdk/wrappers/GroupWrapper.kt

```diff
@@ -2,33 +2,27 @@ package expo.modules.xmtpreactnativesdk.wrappers
 
 import com.google.gson.GsonBuilder
 import org.xmtp.android.library.Client
-import org.xmtp.android.library.Conversation
 import org.xmtp.android.library.Group
-import org.xmtp.android.library.codecs.Attachment
 import org.xmtp.android.library.toHex
-import uniffi.xmtpv3.GroupPermissions
 
 class GroupWrapper {
 
     companion object {
         fun encodeToObj(client: Client, group: Group): Map<String, Any> {
-            val permissionString = when (group.permissionLevel()) {
-                GroupPermissions.ALL_MEMBERS -> "all_members"
-                GroupPermissions.ADMIN_ONLY -> "admin_only"
-                GroupPermissions.CUSTOM_POLICY -> "custom_policy"
-            }
+            val permissionPolicySet = PermissionPolicySetWrapper.encodeToJsonString(group.permissionPolicySet())
             return mapOf(
                 "clientAddress" to client.address,
                 "id" to group.id.toHex(),
                 "createdAt" to group.createdAt.time,
                 "peerInboxIds" to group.peerInboxIds(),
                 "version" to "GROUP",
                 "topic" to group.topic,
-                "permissionLevel" to permissionString,
+                "permissionPolicySet" to permissionPolicySet,
```

Comment:
I'm not sure this is still needed here since you no longer parse it on the group object?

Reply:
good call, this isnt being used anymore, removed here 1e5fcd5281a4e097df196f20f8dbb0022af6ea79